### ISSUE TITLE
ChartBlockMacro weird output

### ISSUE THREAD

**Issue by rahmanusta** [asciidoctor/asciidoctor-extensions-lab#43]
Hi, 

I want to build chart-block-macro, after build I'm getting only

```javascript
/* Generated by Opal 0.6.3 */
(function($opal) {
  var $a, $b, TMP_1, self = $opal.top, $scope = $opal, nil = $opal.nil, $breaker = $opal.breaker, $slice = $opal.slice;

  self.$require_relative("chart-block-macro/extension");
  return ($a = ($b = $scope.Extensions).$register, $a._p = (TMP_1 = function(){var self = TMP_1._s || this, $a;

  if ((($a = self.$document()['$basebackend?']("html")) !== nil && (!$a._isBoolean || $a == true))) {
      return self.$block_macro($scope.ChartBlockMacro)
      } else {
      return nil
    }}, TMP_1._s = self, TMP_1), $a).call($b);
})(Opal);
```

What can be the reason?

**Comment by Mogztter** [comment]
Hi,

I'm also working on cross compiling this extension to JavaScript. See
https://github.com/asciidoctor/asciidoctor.js/issues/97

I made good progress but c3js is not compatible with Opal (c3js is trying
to call bind function on Opal in a deep call stack).

To answer you question, Opal didn't resolve the require so you just have
the code that register the extension.
I made changes to resolve that issue, are you using the latest version
(master up to date) ?

Guillaume
